src/dev/chromium_version.ts

```diff
@@ -7,16 +7,15 @@
  */
 
 import { run } from '@kbn/dev-cli-runner';
-import { ToolingLog } from '@kbn/tooling-log';
```

Comment:
The Chromium team has changed their public APIs on looking up info about Chromium versions and commits. See https://groups.google.com/a/chromium.org/g/chromium-dev/c/uH-nFrOLWtE/m/HW1BoGNPAgAJ